src/main/java/edu/wpi/first/pathui/PathDisplayController.java

```diff
@@ -80,6 +82,17 @@ private void setupPathList() {
         }
       }
     });
+    currentPath.addListener((change, oldValue, newValue) -> {
+      vectorGroup.getChildren().clear();
+      if (newValue == null || newValue == oldValue) {
```

Comment:
Do you mean `==` or `.equals()`?

Reply:
Its the same object so == is fine.

Reply:
`newValue` will never equal `oldValue`, so that comparison is redundant anyway